fix(jans-auth-server): build is failing after smtp PR merge

**Describe the bug**

Build is failing after smtp PR merge.

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:2.3.2:testCompile (default-testCompile) on project jans-auth-model: Compilation failure: Compilation failure: 
[ERROR] /home/jetty/.jenkins/workspace/jans-auth-server/jans-auth-server/model/src/test/java/io/jans/as/model/util/SecurityProviderUtilityTest.java:[16,45] error: cannot find symbol
[ERROR]  class SecurityProviderUtility
[ERROR] /home/jetty/.jenkins/workspace/jans-auth-server/jans-auth-server/model/src/test/java/io/jans/as/model/util/SecurityProviderUtilityTest.java:[23,45] error: cannot find symbol
```

https://jenkins.jans.io/jenkins/job/jans-auth-server/9316/console

--- a/jans-auth-server/model/src/test/java/io/jans/as/model/util/SecurityProviderUtilityTest.java
+++ b/jans-auth-server/model/src/test/java/io/jans/as/model/util/SecurityProviderUtilityTest.java
@@ -13,13 +13,15 @@ public class SecurityProviderUtilityTest extends BaseTest {
     public void installBCProvider_silent_validInstance() {
         showTitle("installBCProvider_silent_validInstance");
         SecurityProviderUtility.installBCProvider(true);
-        assertNotNull(SecurityProviderUtility.getInstance());
+        assertNotNull(SecurityProviderUtility.getBCProvider());
+        assertNotNull(SecurityProviderUtility.getSecurityMode());
     }
 
     @Test
     public void installBCProvider_Nosilent_validInstance() {
         showTitle("installBCProvider_Nosilent_validInstance");
         SecurityProviderUtility.installBCProvider();
-        assertNotNull(SecurityProviderUtility.getInstance());
+        assertNotNull(SecurityProviderUtility.getBCProvider());
+        assertNotNull(SecurityProviderUtility.getSecurityMode());
     }
 }

